app/models/classification.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+# Schema version: 20200509082917
+#
+# Table name: classifications
+#
+#  id              :integer          not null, primary key
+#  info_request_id :integer
+#  state           :string
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#
+
+##
+# A basic Info Request classification status updates
+#
+class Classification < ApplicationRecord
```

Comment:
This is pretty close to [`RequestClassification`](https://github.com/mysociety/alaveteli/blob/develop/app/models/request_classification.rb) so I think we need to consider the additional indirection this addition will cause.

Is it intentional that this doesn't really _do_ anything at the moment? I assume the thinking was that at some point we'll extract `set_described_state` into this on a `before_save` callback?

Do we _need_ to tie classifications to `Project::Submission` at this point? What are we losing if we don't?

Reply:
Not quite sure what to do here yet. I guess we could associate `Project::Submission` to an `InfoRequestEvent` with `event_type=status_update`  but that is ideal either.